app/context-menus.js

```diff
@@ -6,7 +6,18 @@ const {
   clipboard
 } = require('electron')
 
-const path = require('path').posix
```

Comment:
Might want to have a `posixPath` and use that everywhere where `path` was used before.

Reply:
What is posix used for?

Reply:
Done